scripts/build_ngx_pagespeed.sh

```diff
@@ -612,7 +612,8 @@ Not deleting $directory; name is suspiciously short.  Something is wrong."
     nps_downloaded="$TEMPDIR/$nps_downloaded_fname.zip"
     status "Downloading ngx_pagespeed..."
     run wget "$nps_baseurl/$tag_name.zip" -O "$nps_downloaded"
-    nps_module_dir="$BUILDDIR/$nps_downloaded_fname"
+    nps_module_dir=$(unzip -qql "$nps_downloaded" | head -n1 | tr -s ' ' | cut -d' ' -f5-)
```

Comment:
would be nice to add a comment indicating what this pipe transformation is expected to do.